spring-cloud-deployer-local/src/main/java/org/springframework/cloud/deployer/spi/local/DockerCommandBuilder.java

```diff
@@ -45,26 +46,32 @@
 	private final Logger logger = LoggerFactory.getLogger(getClass());
 
 	@Override
-	public String[] buildExecutionCommand(AppDeploymentRequest request, Map<String, String> args, Optional<Integer> appInstanceNumber) {
-		List<String> commands = addDockerOptions(request, args, appInstanceNumber);
+	public String[] buildExecutionCommand(AppDeploymentRequest request, Map<String, String> appInstanceEnv,
+										  Map<String, String> appProperties, Optional<Integer> appInstanceNumber) {
+		List<String> commands = addDockerOptions(request, appInstanceEnv, appProperties, appInstanceNumber);
+		// Add appProperties
+		for (String prop : appProperties.keySet()) {
```

Comment:
The CF deployer has a `spring.cloud.deployer.cloudfoundry.use-spring-application-json` property, maybe we can have the same here and default to `true`